mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price + (price * (20 / 100 ))
+}
```

Comment:
That's correct, in order to calculate the tax, you can multiply the value by 20 and then divide it by 100. And then to find the final price, you can add it to the original price.